src/Timeout/Timeout.php

```diff
@@ -69,14 +69,16 @@ private static function init(): void
     /**
      * Set a timer to throw an exception.
      */
-    public static function timeoutAfter(int $seconds): void
+    private static function timeoutAfter(int $seconds): void
```

Comment:
I tried to make as much as possible `@internal` or private, so that we don't commit to more than necessary regarding the public API

Reply:
This method can be merged with the new `enable()`

Reply:
Yes good point, done!